clients/cobol-lsp-vscode-extension/src/__tests__/services/LanguageClientServiceTest.spec.ts

```diff
@@ -41,8 +41,8 @@ const copyBooksDownloader: CopybookDownloadService = new CopybookDownloadService
 const middleware: Middleware = new Middleware(copyBooksDownloader);
 let languageClientService: LanguageClientService;
 
-const SERVER_DESC = "LSP extension for COBOL language";
-const SERVER_ID = "COBOL";
+const SERVER_DESC = "LSP extension for cobol language";
```

Comment:
This message should stay as is with "LSP extension for COBOL language". It may be needed to fix tests to convert language ID to uppercase.

Reply:
fixed